document_page/__manifest__.py

```diff
@@ -43,7 +32,5 @@
     'demo': [
         'demo/document_page.xml'
     ],
-    'installable': False,
```

Comment:
This is a special case due to the OCA migrate_branch script that was not covered in the regexp expressions, so it could lead to broken manifest files, but I think @StefanRijnhart fixed it when he adapted the script for v10, so it's no more an issue (but needs confirmation by his part).